macroid-core/src/test/scala/macroid/TweakingSpec.scala

```diff
@@ -56,4 +57,16 @@ class TweakingSpec extends FlatSpec {
       val z = lp[LinearLayout](0, 0, 1.0f)
     }
   }
+
+  "TextTweaks.allCaps" should "be invokeable without parenthesis" in {

```

Comment:
Ah, missed a typo here :) Should have been “parentheses”. Unless I forget, will fix this next time I commit.